File: ama/AstroWeather2/app/src/main/java/com/android/blawniczak/astroweather/domain/woeid_city/Place.java
package com.android.blawniczak.astroweather.domain.woeid_city;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Place {

    @SerializedName("name")
    @Expose
    public String name;

    @SerializedName("woeid")
    @Expose
    public String woeid;

}

File: paiwjs/AstroWeather2/app/src/main/java/com/android/blawniczak/astroweather/domain/woeid_city/Place.kt
package com.android.blawniczak.astroweather.domain.woeid_city

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class Place {

    @SerializedName("name")
    @Expose
    var name: String? = null

    @SerializedName("woeid")
    @Expose
    var woeid: String? = null

}
